salt/kubernetes-master/init.sls

```diff
@@ -16,7 +16,9 @@ include:
 {% endfor %}
 
 # add some extra names the API server could have
-{% set extra_names = ["DNS: " + grains['fqdn']] %}
+{% set extra_names = ["DNS: " + grains['fqdn'],
+                      "DNS: api",
```

Comment:
I think it doesn't hurt, and it can be nice to be able to `ping api` from the minions...